docs/content/guides/author-apps/custom/howto-extenders/index.md

```diff
@@ -0,0 +1,78 @@
+---
+type: docs
+title: "How-To: Model any resource with an extender"
+linkTitle: "Extenders"
+description: "Learn how to use extenders in an application"
+weight: 300
+---
+
+This guide will walk you though how to use extenders in an application to model any resource.
+
+## Prerequisites
+
+- [rad CLI]({{< ref "installation#step-1-install-the-rad-cli" >}})
+- [Radius Bicep VSCode extension]({{< ref "installation#step-2-install-the-vs-code-extension" >}})
+- [Radius environment]({{< ref "installation#step-3-initialize-radius" >}})
+
+## Step 1: Register an extender Recipe
+
+In this guide you will leverage a [Recipe]({{< ref "/guides/recipes/overview" >}}) to deploy backing infrastructure for your resource. Begin by registering the 'extender-postgresql' Recipe in your environment:
+
+```bash
+rad recipe register postgresql --resource-type "Applications.Core/extenders" --template-kind bicep --template-path "ghcr.io/radius-project/recipes/local-dev/postgresql:latest"
+```
+
+## Step 2: Define an extender
+
+Open a new file named `app.bicep` and define an extender:
```

Comment:
I know how-to guides are designed to be slimmer in terms of background context, but I think it might be helpful here to briefly mention what this extender provides?

Reply:
Great callout. Added some additional info at the top